src/nextjournal/clerk/render/editor.cljs

```diff
@@ -21,7 +21,7 @@
             [shadow.esm]))
 
 (defn eval-string
-  ([source] (sci/eval-string* (sci.ctx-store/get-ctx) source))
+  ([source] (eval-string (sci.ctx-store/get-ctx) source))
   ([ctx source]
    (when-some [code (not-empty (str/trim source))]
      (try {:result (sci/eval-string* ctx code)}
```

Comment:
@borkdude not sure what's going on here. This first change wraps the result in a `:result` key which is needed by the view, but still doesn't fix the issue. Can you see why ns aliases from evaluating ns forms don't stick in the context?